src/iap/shared/components/game-card/game-card.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+import { Game } from '@shared/models/dto/game.model';
```

Comment:
:point_right: :point_left:
```suggestion
import { Component, Input } from '@angular/core';

import { Game } from '@shared/models/dto/game.model';
```